hybrid/compartment_examples/inter_comp_call/base/Makefile.morello-hybrid

```diff
@@ -3,7 +3,8 @@
 
 SHARED_SOURCES := shared.s
 CFILES := $(wildcard *.c)
-ROOTDIR=../../../..
+ROOTDIR = ../../../..
+CFLAGS += -I$(ROOTDIR)/include -I$(ROOTDIR)/hybrid/include -Iinclude
```

Comment:
Do we need all these three includes?

Reply:
This is a concern since we have so many include files lying around, but that's probably better saved for a future, larger scale refactoring.

Reply:
I switched back to relative includes in https://github.com/capablevms/cheri-examples/pull/76/commits/4ffda3fb3ba60d245ca3144a7f5b9a5e72485ac8